packages/field-base/src/input-controller.js

```diff
@@ -3,14 +3,15 @@
  * Copyright (c) 2021 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { SlotController } from './slot-controller.js';
+import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
 
 /**
  * A controller to create and initialize slotted `<input>` element.
  */
 export class InputController extends SlotController {
   constructor(host, callback) {
-    super(host, [
+    super(
```

Comment:
What do you think about having the following API:

```js
get slotName() {
  return 'input';
}

slotNodeFactory() {
  return document.createElement('input');
}

slotNodeInitializer(node) {
  node.id = this.host.id;
  ...
  this.callback();
}
```

...or:

```js
constructor(host) {
  super(host);

  this.slotName = 'helper';
  this.slotFactory = () => {
    ...
  }
  this.slotInitializer = (node) => {
    ...
  }
}
```

To me, those would be a bit more readable and clear than a constructor with a set of arguments.

Reply:
I would like to avoid changing API in this PR, it's mostly about moving the controller to other package.

Reply:
👍 Good point, we can consider this later.